src/Runner.Listener/JobDispatcher.cs

```diff
@@ -18,6 +18,7 @@ namespace GitHub.Runner.Listener
     [ServiceLocator(Default = typeof(JobDispatcher))]
     public interface IJobDispatcher : IRunnerService
     {
+        bool Busy { get; }
```

Comment:
There are 3 different parts in the runner having Http connection with service.
- Message queue loop for pull job from service.
- Job Dispatcher handle renew lock for running job.
- Runner self-updater handle download latest runner and post update progress to service.

I want to make sure there is no job and self-update running when migrate to use new authorization during message queue loop.